### ISSUE TITLE
LinkedIn Ads/Form trigger

### ISSUE THREAD

**Issue by henrywangnl** [PipedreamHQ/pipedream#8900]
**Name of app / service**
LinkedIn Ads

**Link to developer documentation**
https://learn.microsoft.com/en-us/linkedin/marketing/integrations/lead-sync/leadsync?view=li-lms-2023-07&tabs=http

**Is lack of support preventing you from building workflows, or do you have a workaround?**
Didn't see LinkedIn Ads in Pipedream

**Are there specific actions, or triggers, you'd like to see for this app? Please let us know here or use the Action and Trigger issue templates to open requests for each!**
I would like to have a trigger for new form response from a LinkedIn event. For example, someone signs up for our company's LinkedIn events, I would like to get a trigger and then add it to our HubSpot CRM.

**Comment by sergio-eliot-rodriguez** [comment]
Hi there @henrywangnl 
Thanks for the request.

I'm confused. LinkedIn Ads has already an app integration: https://pipedream.com/apps/linkedin-ads

About that trigger you are describing, does it refers specifically to LinkedIn Ads, or did you mean [LinkedIn Gen Forms](https://business.linkedin.com/marketing-solutions/native-advertising/lead-gen-ads)?

I'd appreciate if you could clarify on this.

Thanks,
Sergio

**Comment by sergio-eliot-rodriguez** [comment]
Hi there @henrywangnl 

Thanks for the request.

And sorry to bug you on this one.

We have LinkdInAds integrated -- https://pipedream.com/apps/linkedin-ads

That said; I noticed in the link to the provided documentation that LeadForms may fall in a distinct program to the Advertising API. This other program is the LeadSync API.

Could you please double check if the current app integration helps your purpose?

**Comment by henrywangnl** [comment]
Hi @sergio-eliot-rodriguez ,

The LinkedIn Ads integration you are referring to doesn't support triggers.
I want to have a trigger for new form responses.

Specifically, here is what I need:
<img width="893" alt="image" src="https://github.com/PipedreamHQ/pipedream/assets/20847528/eb4037dd-0b87-4dfc-a847-f6fc119197ef">

I couldn't find an existing integration on Pipedream. 
Please let me know if you need more information.

**Comment by sergio-eliot-rodriguez** [comment]
Thank you @henrywangnl , that's all needed. Seeing now how to move forward in this app integration for LinkedIn Froms.

**Comment by sergio-eliot-rodriguez** [comment]
@malexanderlim can't use the linkedin account tied to my pipekit.pro email, LinkedIn requires the company information be verified to apply for the Lead Sync API (needed in this use case). when you are back let's revisit, and maybe i can help tackling the oauth flow with PD credentials.

![image](https://github.com/PipedreamHQ/pipedream/assets/101526248/44184f76-a377-45f5-8b07-4f5302047d44)

**Comment by sergio-eliot-rodriguez** [comment]
Hi @henrywangnl 

Happy to announce that his has been integrated

Check it out at https://pipedream.com/apps/linkedin-ads?via=go

and let us know any feedback !

**Comment by henrywangnl** [comment]
Hi @sergio-eliot-rodriguez ,

Thanks for your awesome work and now I'm one step closer to switch completely from Zapier to Pipedream ;-)
One question about the pulling frequency tho:
<img width="927" alt="image" src="https://github.com/PipedreamHQ/pipedream/assets/20847528/ed907b4a-e370-45c0-8075-0a3e2ee4533f">

Why do we have to search for new registration instead of get notification of new registration?
I'm asking because there is a scenario where registrants would expect to get confirmation emails after signup so the user experience is not good if the frequency is too high (every 15 minutes etc). On the other hand, if the frequency is too low (30 seconds), would it affect my credits on Pipedream? The last concern is, this workflow may keep running even after an event on LinkedIn is over, unless I remember to pause it in time.

To sum up, getting notifications of new registrations would be preferable in this case. Not sure if that's possible from the api perspective, but that's the case on Zapier.
Hope it helps!

**Comment by malexanderlim** [comment]
Hi @henrywangnl , totally understood the desire to use webhooks here. We're awaiting approval from the LinkedIn team for Standard Access, that would allow us to build out webhook functionality for LinkedIn Ads. Hoping to get this approved in the coming weeks to solve for this use case!
